src/app/js/public/facet/sagas.js

```diff
@@ -37,11 +38,20 @@ export function* clearFacetSaga({ payload: { action } }) {
     yield put(clearFacet());
 }
 
+const scrollToTop = () =>
+    window.scroll({ top: 0, left: 0, behavior: 'smooth' });
```

Comment:
is it the same used in src/app/js/Root.js?

Also, please check the existence of `window` before calling `scroll` - this will ease unit tests in Node.js

Reply:
It is not the same since here I need a smooth animation, whereas on route change I need it to be instant. But it still could be refactored into a single helper function :)